src/admin-panel/utils.js

```diff
@@ -0,0 +1,10 @@
+function createElement({ type, attributes = {}, innerText }) {
+  const element = document.createElement(type);
+  Object.keys(attributes).forEach((item) => {
+    element.setAttribute(item, attributes[item]);
+  });
+  element.textContent = innerText;
+  return element;
+}
+
+export { createElement };
```

Comment:
you have this same piece of code in the production, can't we make this into a generic util function and then import it her
Reference - https://github.com/Real-Dev-Squad/website-dashboard/blob/develop/src/taskEvents/utils.js#L3-L10

Reply:
I think, this can be done in another PR, created an issue for it, we can assign it to any beginner https://github.com/Real-Dev-Squad/website-dashboard/issues/252